test/log-generators/pipelines/install.sh

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
```

Comment:
Hey @ggallen could you add the license lines here, please?
`# Copyright KubeArchive Authors

# SPDX-License-Identifier: Apache-2.0
`

Reply:
Fixed.!